draft-reddy-cose-jose-pqc-hybrid-hpke.md

```diff
@@ -113,6 +113,40 @@ For readability the algorithm ciphersuites labels are built according to the fol
 
 The HPKE PQ/T hybrid ciphersuites for JOSE and COSE are defined in {{IANA}}. Note that the PQ/T Hybrid KEM in HPKE is not an authenticated KEM. The HPKE Base mode can only be supported with the PQ/T Hybrid KEM.
 
+
+# AKP Key for X-Wing 
+
+This section describes the required parameters for an "AKP" key type, as defined in {{!I-D.ietf-cose-dilithium}}, and its use with the X-Wing algorithms, as defined in {{IANA}}. An example JWK is also provided for illustration.
```

Comment:
I would use this references for "X-Wing algorithms" instead of IANA.

https://datatracker.ietf.org/doc/draft-connolly-cfrg-xwing-kem/

Reply:
Instead of repeating the definition of AKP (possibly incorrectly), this should state what "pub" and "priv" are for these algorithms.